ldk/javascript/src/whisper/types.ts

```diff
@@ -233,6 +236,12 @@ export type Message = WhisperComponent<WhisperComponentType.Message> & {
   tooltip?: string;
 };
 
+export type SectionTitle = WhisperComponent<WhisperComponentType.SectionTitle> & {
+  body: string;
+  textAlign?: TextAlign;
+  backgroundStyle?: string;
```

Comment:
Why not use enum instead, so loop author dont have to guess what values this could take?

Reply:
@RusGraf  Yeah, we can do enum！

Reply:
I was like, there's only 2 options, so I just used string there